public/app/features/explore/Logs.tsx

```diff
@@ -353,12 +353,12 @@ export class UnthemedLogs extends PureComponent<Props, State> {
           </div>
           <LogsNavigation
             logsSortOrder={logsSortOrder}
-            visibleRange={visibleRange}
+            visibleRange={visibleRange || absoluteRange}
```

Comment:
```suggestion
            visibleRange={visibleRange ?? absoluteRange}
```

i recommend the `??` operator here, it explains better the i-want-to-use-absoluteRange-when-that-is-undefined (i did not test the change though)